pyttb/sumtensor.py

```diff
@@ -108,9 +108,9 @@ def __repr__(self):
             tensor of shape (2, 2)
             data[:, :] =
             [[1. 1.]
-            [1. 1.]]
+             [1. 1.]]
```

Comment:
The NORMALIZE_WHITESPACE ignores tabs and spaces. Since you were only indenting with `\t` before each part, I fixed the indentation that `numpy` provides for the `tensor.data` objects. This now reflects the output where only the `\t` is normalized.